internal/cli/cmd/sysdump.go

```diff
@@ -34,9 +34,9 @@ var (
 
 func newCmdSysdump() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:    "sysdump",
-		Short:  "Collects information required to troubleshoot issues with Cilium and Hubble",
-		Long:   ``,
+		Use:   "sysdump",
+		Short: "Collects information required to troubleshoot issues with Cilium and Hubble",
+		Long:  ``,
```

Comment:
It's really weird that the test passed in the original PR. Could it be due to a bug in golangci-lint when setting this to true? https://github.com/cilium/cilium-cli/blob/6a1699d6f34f4b65d333b41b01999ee70db35f47/.github/workflows/go.yml#L35-L36

Reply:
Ah, just saw the second commit and seems you had the same though as I did :smiley: